about.html

```diff
@@ -0,0 +1,137 @@
+                <a class="navbar-main__link" href="#staff">Our Instructors</a>
+              </li>
+              <li><a class="navbar-main__link" href="#">FAQ</a></li>
+            </ul>
+            <button id="cc-campaign" type="button">Book Now!</button>
+            <div class="menu-end">
+              <hr />
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section id="about-hero">
+      <div class="flex-container">
+        <h2>"Hello Aspiring Thrillers!"</h2>
+        <h1>About Air Sports Club</h1>
+        <div class="about-hero__message">
+          <p>
+            ​At Air Sports Club, we specialize in small group for multiple air
+            sport adventures. Over the past 28 years, Air Sports Club has grown
+            rapidly to become one of the leading sport club like adventure
+            companies. We Provide Instructors to help you do the air actively
+            and explore the adventure of Skydiving, Paragliding,
+            Paramotoring,Hang Gliding, Wingsuit Flying and more. We always dig
+            deeper in our new destinations to keep evolving our new locations or
+            spot with the core intention of keeping your experiences truly
+            original and the memories priceless. When you are doing this
+            activities, you just feel more Thrilled. When you see more
+            activities, you meet more people, create more lasting memories...
+            When you trust in our expertise to deliver you these extraordinary
+            experiences, it truly means the world to us.
+          </p>
+        </div>
+        <h4 class="about-hero__contact"></h4>
+          Please contact us on the email below for any further queries <br />
+          about our services.
+        </h4>
+        <a class="about-hero__email" href="#Contact"
+          >ashwincarvalho27@gmail.com</a
+        >
+      </div>
+    </section>
+    <section id="about-logo">
+      <div class="flex-container">
+        <h2>Air Sports Club Logo</h2>
+        <hr class="section__hr" />
+        <p>
+          The Air Sports Club logo was decided after a logo design competition
+          from 5th of June to 7th of July during the company logo contest, even
+          the friends of the Instructors took part and had great fun in the logo
+          contest.
+        </p>
+        <div class="about-logo__show">
+          <img src="assets/images/air-sports-logo.jpg" alt="Air Sports Club" />
+        </div>
+      </div>
+    </section>
+    <section id="about-past">
+      <div class="flex-container">
+        <h2>Recent Sports Events</h2>
+        <hr class="section__hr" />
+        <p>
+          Take a look at the last two Sports Events which took place in Jan 2021
+          and Feb 2021
+        </p>
+        <div class="about-past__grid">
+          <article class="about-past__cards card1"></article>
+          <article class="about-past__cards card2"></article>
+        </div>
```

Comment:
Kindly add a centered text over these two images as shown in the template design 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/190355948-1eb1dad8-52ab-45fa-9dd3-63e87a4d81d1.png)

Template design:

![image](https://user-images.githubusercontent.com/73714615/190356040-ef798207-c0b0-4564-ab28-e0d5d1c23876.png)